### PR TITLE
Docs: Revising Get Started Content

### PR DESCRIPTION
**Pull request by RounakDhillon**
### Describe your changes:

Fixes <issue-number>

I worked on updating get started content as per feedback from Prajwal.

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by Prajwal214 — DISMISSED** [review]
Approved

File: openmetadata-docs/content/v1.6.x-SNAPSHOT/how-to-guides/getting-started/day-1/index.md [hunk 1]
```diff
@@ -83,21 +83,35 @@ to import metadata into OpenMetadata on a regular schedule.
...
+
+### Inviting Users
 
 After metadata has been ingested into OpenMetadata, you can [invite users](/how-to-guides/admin-guide/teams-and-users/invite-users) to collaborate on the data and assign different roles.
 
 - Go to **Settings > Team & User Management > Users**.
 
 {% image
-  src="/images/v1.6/getting-started/users.png"
+  src="/images/v1.5/getting-started/users.png"
```
Comment (ayush-shah) [inline openmetadata-docs/content/v1.6.x-SNAPSHOT/how-to-guides/getting-started/day-1/index.md:107]: Let's revert this change, the images to should refer to the version the content is on. 1.6 content should have images from 1.6

**Review by ayush-shah — CHANGES_REQUESTED** [review]

File: openmetadata-docs/content/v1.6.x-SNAPSHOT/how-to-guides/getting-started/day-1/index.md (hunk 1 above)
Comment (RounakDhillon) [reply openmetadata-docs/content/v1.6.x-SNAPSHOT/how-to-guides/getting-started/day-1/index.md:107]: ok

**Review by Prajwal214 — APPROVED** [review]
Approved
